packages/rx_bloc/lib/src/result_extensions.dart

```diff
@@ -35,11 +36,12 @@ extension ResultMapStreamX<E> on Stream<E> {
   /// ```
   Stream<Result<R>> withLatestFromResult<R>(
     Stream<Result<R>> stream,
-    R Function(E, R) mapSuccess,
-  ) {
+    R Function(E, R) mapSuccess, {
+    String? tag,
+  }) {
     return withLatestFrom<Result<R>, Result<R>>(
```

Comment:
Maybe here can be used the other shorthand syntax:
```=>
     withLatestFrom<Result<R>, Result<R>>(
      stream,
      (e, t) => t.mapResult((data) => mapSuccess(e, data), tag: tag),
    );```

Reply:
Done